api/bot/slackbot/v1/slackbot.proto

```diff
@@ -0,0 +1,106 @@
+syntax = "proto3";
+
+package clutch.bot.slackbot.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/bot/slackbot/v1;slackbotv1";
+
+import "api/v1/annotations.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/struct.proto";
+
+service SlackBotAPI {
+  rpc Event(EventRequest) returns (EventResponse) {
+    option (google.api.http) = {
+      post : "/v1/bot/slackbot/event"
+      body : "*"
+    };
+    option (clutch.api.v1.action).type = CREATE;
+  }
+}
+
+// Information on the bot user
+message Bot {
+  // bot id
+  string id = 1;
+  // if the bot is deleted
+  bool deleted = 2;
+  // bot name
+  string name = 3;
+  // timestamp of when the bot app was last updated
+  int64 updated = 4;
+  // unique identifier of the installed Slack application
+  string app_id = 5;
+  // image urls
+  map<string, string> icons = 6;
+  // unique identifier of the workspace where the event occurred
+  string team_id = 7;
+}
+
+// The structure of events vary among types. Full list of event types: https://api.slack.com/events.
+// We will be receiving app_mention events (events that mention the bot) or message.im events (a message posted in a DM
+// with the bot).
+message Event {
+  // the type of event
+  string type = 1;
+  // the user who messaged the bot
+  string user = 2;
+  string bot_id = 3;
+  Bot bot_profile = 4;
+  // the message text
+  string text = 5;
+  // timestamp
+  string ts = 6;
+  // the channel where the event happened
+  string channel = 7;
+  // for DM messages, this will be "im"
+  string channel_type = 8;
+  // when the event was dispatched
+  string event_ts = 9;
+  string client_msg_id = 10;
+  // unique identifier of the workspace where the event occurred
+  string team = 11;
```

Comment:
is this the same as `team_id` above?

Reply:
yep it's the same as the `team_id` in the outer object that the API sends us. in the event object it's named `team`. Will add a comment that it's an identical value as `team_id`